data/strings/strings.txt

```diff
@@ -6440,7 +6440,7 @@
     ca = Notícies
     cs = Novinky
     da = Nyheder
-    de = News
+    de = Neuigkeiten
```

Comment:
In the past, the original german term was deliberateley changed to the english, yet common term "News".

Reply:
Then I'll let it be as "News"